app/models/concerns/orderable.rb

```diff
@@ -2,9 +2,16 @@ module Orderable
   extend ActiveSupport::Concern
 
   included do
-    default_scope { 
-      order(featured: :desc, comments_count: :desc, created_at: :desc, id: :desc) 
+    scope :order_by, lambda{ |field|
+      if (field)
```

Comment:
Nope, should be all good! The default_scope gets applied to everything unless specifically "unscoped". So if `!field.present?` then `unscope(:order)` won't happen, and default_scope's ordering will be maintained.